src/System.Security.Cryptography.Xml/tests/EndToEnd.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+using Xunit;
+
+namespace System.Security.Cryptography.Xml.Tests
+{
+    public class EndToEnd
```

Comment:
Dare I be really fussy, this should either have a more descriptive class name talking about the tests it performs or be parameterized to take different transforms. From memory, this also overlaps with existing tests in SignedXml. If we accept this, the test maybe subsumed into other tests, anyway.